pom.xml

```diff
@@ -246,7 +245,7 @@
       <!-- End publish to maven central -->
 
       <!-- Begin source & javadocs being generated -->
-      <plugin>
+      <!-- <plugin>
```

Comment:
Reminder that we'll want to uncomment these here and elsewhere.

Reply:
ugh. I didn't mean to commit this.

I have an issue with this pom and Java 18. I was able to resolve it in the contribs.

Reply:
reverted.